hack/e2e.go

```diff
@@ -19,51 +19,47 @@ package main
 
 import (
 	"flag"
+	"fmt"
+	"github.com/sergi/go-diff/diffmatchpatch"
```

Comment:
I think the original goal of `e2e.go` was to not require any external dependencies, but that's probably not useful anymore.